src/components/cylc/cylcObject/Menu.vue

```diff
@@ -135,17 +144,56 @@ export default {
     this.$eventBus.off('show-mutations-menu', this.showMutationsMenu)
   },
 
+  computed: {
+    canExpand () {
+      if (!this.mutations) {
+        return false
+      }
+      if (this.$workflowService.primaryMutations[this.type]) {
+        return true
+      }
+      return false
+    },
+
+    displayMutations () {
+      if (!this.mutations) {
+        return []
+      }
+      const shortList = this.$workflowService.primaryMutations[this.type]
+      if (!this.expanded && shortList) {
+        return this.mutations.filter(
+          (x) => {
+            return shortList.includes(x[0].name)
+          }
+        ).sort(
+          (x, y) => {
+            return shortList.indexOf(x[0].name) - shortList.indexOf(y[0].name)
+          }
```

Comment:
So default order is alphabetical, but `primaryMutations` list can specify another order for the mutations to be displayed :+1: